src/main/kotlin/minesweeper/dto/HeightResult.kt

```diff
@@ -0,0 +1,16 @@
+package minesweeper.dto
+
+import minesweeper.utils.Validation
+
+class HeightResult(height: String) {
+
+    var height: Int
+        private set
+
+    init {
+        require(height.isNotEmpty()) { "공백이 들어옴" }
+        require(Validation.isNumeric(height)) { "숫자가 아닌 문자가 들어옴" }
+        this.height = height.toInt()
+    }
```

Comment:
```suggestion
class HeightResult(val height: Int) {

     companion object {
            fun of(height: String): Height {
                 require(height.isNotEmpty()) { "공백이 들어옴" }
                 require(Validation.isNumeric(height)) { "숫자가 아닌 문자가 들어옴" }
                 return Height(height.toInt())
            }
     }
```

저는 Height 로써 Height에 대한 Int 값만 가지고 있을 수 있도록 변경할 것 같아요. 그리고 이에 대해서 다른 타입이 온다면 정적 팩토리 메서드를 통하여 표현을 해줄 것 같습니다. 그 이유는 클래스에서 가지고 있는 프로퍼티에 대해서는 타입을 명확히 가지고 있는 게 유지보수와 확장에 대해서 많은 이득을 낼 수 있다고 생각하기 때문입니다.